Jenkinsfile.deploy

```diff
@@ -171,12 +171,53 @@ pipeline {
             }
         }
 
-        stage('Deploy artifacts') {
+        stage('Deploy to repository') {
+            when {
+                expression { return !shouldStageArtifacts() && !isSpecificArtifactsUpload() && shouldDeployToRepository() }
+            }
+            steps {
+                script {
+                    runMavenDeploy(getOptaplannerMavenCommand(), params.SKIP_TESTS)
+                    runMavenDeploy(getOptawebVehicleRoutingMavenCommand(), params.SKIP_TESTS)
+                    runMavenDeploy(getOptawebEmployeeRosteringMavenCommand(), params.SKIP_TESTS)
+                }
+            }
+        }
+        stage('Deploy locally') {
+            when {
+                expression { return shouldStageArtifacts() || isSpecificArtifactsUpload() }
```

Comment:
When `shouldStageArtifacts()` is `true`, both this stage and the `Stage artifacts` will run. Is that intentional?

Reply:
yes, because we stage all artifacts which were deployed in local folder so we need this :)